proto/elys/commitment/query.proto

```diff
@@ -0,0 +1,26 @@
+syntax = "proto3";
+package elysnetwork.elys.commitment;
+
+import "gogoproto/gogo.proto";
+import "google/api/annotations.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "elys/commitment/params.proto";
+
+option go_package = "github.com/elys-network/elys/x/commitment/types";
+
+// Query defines the gRPC querier service.
+service Query {
+  // Parameters queries the parameters of the module.
+  rpc Params(QueryParamsRequest) returns (QueryParamsResponse) {
+    option (google.api.http).get = "/elys-network/elys/commitment/params";
+  }
```

Comment:
I think more queries should be available other than params to check the status of the users' deposit.